nautobot/extras/models/jobs.py

```diff
@@ -334,6 +334,101 @@ def get_absolute_url(self):
         return reverse("extras:job_detail", kwargs={"slug": self.slug})
 
 
+@extras_features("graphql")
+class JobHook(OrganizationalModel):
+    """
+    A job hook defines a request that will trigger a job hook receiver when an object is created, updated, and/or
+    deleted in Nautobot. Each job hook can be limited to firing only on certain actions or certain object types.
+    """
+
+    content_types = models.ManyToManyField(
+        to=ContentType,
+        related_name="job_hooks",
+        verbose_name="Object types",
+        limit_choices_to=FeatureQuery("webhooks"),
```

Comment:
Was this intentional in that we want job hooks to be eligible for the same models as webhooks? Just wanted to assert that this wasn't overlooked!

Reply:
I wanted feedback on this and changed it to look for all models that are eligible for change logging